jekyll/_cci2/api-developers-guide.md

```diff
@@ -348,3 +348,73 @@ job -
 
 
 ## Download Artifacts
+
+The following section details the steps you need to follow to download artifacts generated when a job is run. We will go through the steps of returning a list of artifacts for a job, and then changing the command we run to download the full set of artifacts. If you are looing for instructions to download the latest artifacts for a pipeline, without needing to specify a job number, see our [API v1.1 guide](https://circleci.com/docs/2.0/artifacts/#downloading-all-artifacts-for-a-build-on-circleci) – keep checking back here as this functionality will be added to API v2 in the future.
+
+### Prerequisites
+
+Before making an API call, make sure you have met the following prerequisites:
+
+* You have set up a GitHub or BitBucket account with a repository to use with CircleCI.
+* You have completed CircleCI onboarding and you have a project setup.
+* You have a [personal API token](https://circleci.com/account/api).
+* You have been [authenticated](#get-authenticated) to make API calls to the server.
+
+### Steps
+
+1. First we will ensure your api token is set as an environment variable. You maybe have already done this during authentication, but if not, run the following substituting your personal api token:
+
+    ```
+    export CIRCLECI_TOKEN={your_api_token}
+    ```
+
+2.  Next we need the job number for the job you want to get artifacts for. You can find job numbers in the UI - either in the breadcrumbs on the Job Details page, or in the URL.
+
+    INSERT SCREENSHOT HIGHLIGHTING JOB NUMBER
+
+3.  Next we can use `curl` to return a list of artifacts for a specific job. Note, you can format JSON responses by piping the `curl` command into the `jq` utility if you have it installed: `curl ... | jq`:
+
+    ```sh
+    curl -X GET https://circleci.com/api/v2/project/{VCS}/{org-name}/{repo-name}/{job-number}/artifacts \
+    -H 'Content-Type: application/json' \
+    -H 'Accept: application/json' \
+    -H "Circle-Token: $CIRCLECI_TOKEN" | jq\
+    ```
+
+    Check the table below for help with formatting your API call correctly
+
+    Parameter | Description
+    --- | ---
+    `VCS` | Version Control System - either `github`/`gh` or `bitbucket`/`bb`
+    `org-name` | Organization name, or your personal username to your VCS
+    `repo-name` | Name of your project repo
+    `job-number` | The number for the job you want to download artifacts from - see step 2
```

Comment:
Most tables in our docs use a striped class, which you can add, as below:

```
    | Parameter      | Description                                                             |
    | ---------------| ------------------------------------------------------------------------|
    | `VCS`          | Version Control System - either `github`/`gh` or `bitbucket`/`bb`       |
    | `org-name`     | Organization name, or your personal username on your VCS                |
    | `repo-name`    | Name of your project repo                                               |
    | `job-number`   | The number for the job you want to download artifacts from - see step 2 |
    {: class="table table-striped"}
```

Reply:
Ah I think there's quite a bit of different table formatting around docs because I copied this from the config reference I think. I will update